src/data-model/actor-data/rqgActorData.ts

```diff
@@ -14,6 +14,7 @@ export interface CharacterDataSourceData {
   // --- Derived / Convenience Data Below ---
   extendedName: string;
   attributes: Attributes; // Most are derived // TODO Split / move data?
+  editMode: boolean;
```

Comment:
Just formatting, but both `editMode` and `extendedName` should be above the comment `// --- Derived / Convenience Data Below ---` since they are not derived

Reply:
Done